include/functions.php

```diff
@@ -1,55 +1,67 @@
-            ),
-            3=>array(
-                "PostId"=>3,
-                "Title"=>"My Thoughts on the Weather",
-                "Body"=>"<h2>I don't like it when it's too hot. I also don't like it when it's too cold.
-                    Despite my allergies and ashtma, I LOVE being outdoors. When it is warm out I like to go on walks in the park. When the weather is perfect (about 75 degrees) the best place for me to be is at the Boat House in Forest Park or in a canoe on Creve Coeur lake.
-                The outdoors have become a place where I find tranquilty.</h2>"
-            )
-        );
+            (future goals for site and maybe self)</p> </h2>"
+        ),
+        1=>array(
+            "PostId"=>1,
+            "Title"=>"Lets Get Healthy!!",
+            "Body"=>"<h2> First I want to start by saying that our specific details of what it means to be healthy can be different.
+            In fact, they should be different! But we are all striving to be healthy mentally, physically, and emotionally.</p>"
+        ),
+        2=>array(
+            "PostId"=>2,
+            "Title"=>"Treat Yo Self",
+            "Body"=>"<h2>Too often I find myself cutting out things that I enjoy because I have other commitments. So I have been trying
+            to get away from that habit because I would fins that I had not done anything for myself in 2 or 3 weeks! There is a lot of
+            value in spending time with yourself and doing things for yourself. Nothing is more important in your life than you. I mean that.
+            Without you, you could not do ANY of the things that you do. SO take care of you! Take yourself out, walk in the park, take a nap,
+            whatever you want to do. I challenge you to take 3 hours out of your week to do things that make you feel good.</p>"
+        ),
+        3=>array(
+            "PostId"=>3,
+            "Title"=>"My Thoughts on the Weather",
+            "Body"=>"<h2>I don't like it when it's too hot. I also don't like it when it's too cold.
+                Despite my allergies and ashtma, I LOVE being outdoors. When it is warm out I like to go on walks in the park. When the weather is perfect (about 75 degrees) the best place for me to be is at the Boat House in Forest Park or in a canoe on Creve Coeur lake.
+            The outdoors have become a place where I find tranquilty.</h2>"
+        )
+    );
         return $Post;
     }
-// "html".php."html".php;
-    function getAllPost() {
-        $Post = getPosts();
-        foreach($Post as $indiv){
-            // $indiv['title']
-            echo "
-            <a href='view_post.php?PostId=".$indiv['PostId']."'>".$indiv['Title']."</a>";
-        }
-    }
 
     function getPost($PostId) {
-        $Post = getPosts();
-        echo $Post[$PostId]['Title'];
-        echo $Post[$PostId]['Body'];
-
+        $result=dbQuery("
```

Comment:
I wouldn't normally comment on this, but since you said you're working on indentations, it looks like there are still some functions like this one and the next where the initial function definition line is indented when it doesn't need to be, so the whole thing is an extra indent deep.